holidays/locale/uk/LC_MESSAGES/HU.po

```diff
@@ -0,0 +1,109 @@
+# Hungary holidays uk localization.
+# Authors: ~Jhellico <jhellico@gmail.com>, (c) 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Python Holidays 0.27\n"
+"POT-Creation-Date: 2023-06-12 19:06+0300\n"
+"PO-Revision-Date: 2023-06-12 20:10+0300\n"
+"Last-Translator: ~Jhellico <jhellico@gmail.com>\n"
+"Language-Team: Python Holidays localization team\n"
+"Language: uk\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<12 || n%100>14) ? 1 : 2);\n"
+"Generated-By: pygettext.py 1.5\n"
+"X-Generator: Poedit 3.2.2\n"
+
+#. Day off before
+#: ./holidays/countries/hungary.py:45 ./holidays/countries/hungary.py:72
+#, c-format
+msgid "%s előtti pihenőnap"
+msgstr "%s (вихідний)"
```

Comment:
I was unable to find good equivalents for "Day off before/after", considering the problem of cases. So I left one option for both.

Reply:
This option is also okay 👍